src/cmake/ov_parallel.cmake

```diff
@@ -242,7 +242,7 @@ macro(ov_find_package_tbb)
     endif()
 endmacro()
 
-function(set_ie_threading_interface_for TARGET_NAME)
+function(set_ov_threading_interface_for TARGET_NAME)
```

Comment:
maybe we need to give a better name like `ov_set_threading_interface_for`

Reply:
Fixed